packages/react/src/experimental/Information/Avatars/AvatarList/index.tsx

```diff
@@ -73,6 +74,12 @@ type Props = {
    * The remaining number to display.
    */
   remainingCount?: number
+
+  /**
+   * Whether to fill the container with avatars.
+   * @default false
+   */
+  fillContainer?: boolean
```

Comment:
what do you think about having this prop a string with possible layouts? 

For example, `layout: "default" | "fill"`. The default is a bad name and we can come up with a clear one. This will keep the API size untouched in case we need to add another layout for the component